### DISCUSSION TITLE
PKGi PSP v0.8.0

### DISCUSSION THREAD

**Discussion by bucanero** [bucanero/pkgi-psp#3] [Announcements]
First public release. In memory of Leon & Luna.

![leon-luna](https://github.com/bucanero/pkgi-psp/raw/main/docs/leon-luna.jpg)

### Added

* Download and install PKG files to the PSP
* Support for loading multiple database files
  - Generic text database format support
  - Filter unsupported or missing URLs when loading a database
* Content categorization and filtering
* Support for HTTP, HTTPS, FTP, FTPS links with TLS v1.2
* Localization support (Finnish, French, German, Indonesian, Italian, Polish, Portuguese, Spanish, Turkish)
  - Language detection based on PSP settings
* Enter button detection (`cross`/`circle`)

<hr /><em>This discussion was created from the release <a href='https://github.com/bucanero/pkgi-psp/releases/tag/v0.8.0'>PKGi PSP v0.8.0</a>.</em>

**Comment by wilowfrien** [comment]
Buenas,ante todo muchas gracias por el aporte tan maravilloso, algun requisito para que los juegos arranquen? No me arrancan tira un error
![1696187428376228212588488049067](https://github.com/bucanero/pkgi-psp/assets/146670148/d1052bc7-4620-40db-a0af-023e0dde9670)

**Reply by bucanero** [reply]
Es un error de licencias, los contenidos necesitan ser activados. Algunos CFWs y plugins ofrecen alternativas, fijate las notas aqui:
https://github.com/bucanero/pkgi-psp#notes

**Reply by wilowfrien** [reply]
Muchas gracias por su atencion,pude solucionarlo con su respuesta ahora si me funcionan los juegos.

**Reply by Eityrus** [reply]
Como se mete en la consola la licencia? Es que no sé cómo donde se mete ni consigo que lea los archivos.

**Comment by Mouzinhoportugal** [comment]
Congratulations

**Comment by DJTOMATO** [comment]
I don't know why but the app just says corrupted data 6.61 pro-c inf

**Comment by elbhz** [comment]
I'm on ARK 4.20.64 infinity, the app loads to the error PKGi.txt files missing, screen, but the buttons are all unresponsive.

**Reply by bucanero** [reply]
You need to provide your own database files. See details here: https://github.com/bucanero/pkgi-psp#setup-instructions

**Comment by Eityrus** [comment]
No consigo hacer que se me abran los juegos, ¿Como instalo las licencias? No consigo saber dónde van ni consigo que me la lea. ¿podría ayudarme porfavor?

**Reply by bucanero** [reply]
https://github.com/bucanero/pkgi-psp/discussions/3#discussioncomment-7160022

**Comment by Camilooo1637** [comment]
Hola 
![17062356500841036415962491265168](https://github.com/bucanero/pkgi-psp/assets/157772129/063c201c-e703-46ab-8947-adc5eccc0d86)
Porque me aparece eso , con todos los juegos

**Reply by bucanero** [reply]
Update to latest version v1.0.0, be sure you are connected to Wifi, and try again
https://github.com/bucanero/pkgi-psp/releases/tag/v1.0.0

**Reply by Camilooo1637** [reply]
Ohhh , y como instalo esa nueva versión, la verdad nose

**Reply by DJTOMATO** [reply]
ocurre incluso en la version 1.0.0 conectado a wifi
